toqito/nonlocal_games/nonlocal_game.py

```diff
@@ -153,20 +176,22 @@ def classical_value(self) -> float:
             ) = pred_mat_copy.shape
         pred_mat_copy = np.transpose(pred_mat_copy, (0, 2, 1, 3))
 
-        for i in range(num_alice_outputs**num_bob_inputs):
-            number = i
-            base = num_bob_outputs
-            digits = num_bob_inputs
-            b_ind = np.zeros(digits)
-            for j in range(digits):
-                b_ind[digits - j - 1] = np.mod(number, base)
-                number = np.floor(number / base)
-            pred_alice = np.zeros((num_alice_outputs, num_alice_inputs))
+        num_iterations = num_alice_outputs**num_bob_inputs
+
+        if num_iterations > 1000:
```

Comment:
It may be helpful to have a comment above the `if` and `else` statements stating when/why each of these gets invoked.

Reply:
added. Done!